pkg/models/observer_buffer.go

```diff
@@ -44,6 +44,9 @@ type ObserverBuffer struct {
 	MaxFilterLatency    time.Duration
 	MinFilterLatency    time.Duration
 	SumFilterLatency    time.Duration
+	MaxRequestLatency   time.Duration
+	MinRequestLatency   time.Duration
+	sumRequestLatency   time.Duration
```

Comment:
not sure if the sum will tell data that we will be able to reason about, average would for sure be good to reason, but the sum will depend on how many request were made.

Reply:
Yeah, this stat was originally just a hacky thing I used for testing but proved to sometimes be worth having in prod - longer term it deserves design consideration.

For our current purposes, we always have a batch of 1, so sum will give us the ability to compute average regardless.

Reply:
That is to say, sumRequestLatency/(msgSent + msgFailed) = average request latency for the time period.

Reply:
gotcha